rent/src/lib.rs

```diff
@@ -52,16 +64,37 @@ pub struct Rent {
 /// - $1 per SOL
 /// - $0.01 per megabyte day
 /// - $3.65 per megabyte year
+#[deprecated(
+    since = "3.1.0",
+    note = "The concept of rent no longer exists, only rent-exemption. Use `DEFAULT_LAMPORTS_PER_BYTE` instead"
+)]
 pub const DEFAULT_LAMPORTS_PER_BYTE_YEAR: u64 = 1_000_000_000 / 100 * 365 / (1024 * 1024);
 
+/// Default rental rate in lamports/byte.
+///
+/// This calculation is based on:
+/// - 10^9 lamports per SOL
+/// - $1 per SOL
+/// - $0.01 per megabyte day
+/// - $7.30 per megabyte
+pub const DEFAULT_LAMPORTS_PER_BYTE: u64 = 2 * 1_000_000_000 / 100 * 365 / (1024 * 1024);
```

Comment:
SIMD-0194 specifies this value as `6960`, which is equivalent to doubling the current `3480` value. Because of the precision, `DEFAULT_LAMPORTS_PER_BYTE` is evaluating to `6961`. To match the SIMD value, we could do the following:
```suggestion
pub const DEFAULT_LAMPORTS_PER_BYTE: u64 = 2 * (1_000_000_000 / 100 * 365 / (1024 * 1024) as u64);
```